cloudhub-2/modules/ROOT/pages/ps-gather-setup-info.adoc

```diff
@@ -385,6 +405,17 @@ You cannot use the following reserved CIDR blocks:
 169.254.169.252/30
 ----
 
+Anypoint Platform has a list of CIDRs as an internal CIDR pool for provisioning private space networks. The MuleSoft CIDR pool is subject to change. You can configure the following as reserved CIDR blocks, but you cannot use them as private space network CIDRs:
```

Comment:
can we change to "You can configure some of the following as reserved CIDR blocks"?

Reply:
```suggestion
Anypoint Platform has a list of CIDRs as an internal CIDR pool for provisioning private space networks. The MuleSoft CIDR pool is subject to change. You can configure some of the following as reserved CIDR blocks, but you cannot use them as private space network CIDRs:
```